mintme/src/ui/Card/MasonryCard.tsx

```diff
@@ -97,6 +97,7 @@ export const MasonryCard = ({ nftModel, reExecuteQuery, hidePopUp, nft }: Masonr
   const file = nftModel?.content?.files?.[0]
   const poster = nftModel?.content?.poster
 
+  const mounted = useRef(false)
```

Comment:
Can you describe the intent of this change? Is it to re execute query when we mount this component (e.g. when we navigate to this page?)

Reply:
Ah I see

Reply:
Can you add this as a comment in the code as well?